src/PageContainer/component.js

```diff
@@ -233,29 +233,27 @@ class PageContainer extends Component<DefaultProps, Props, void> {
     const LayoutFallback = getLayout(props.defaultLayout, props)
     const Layout = getLayout(page.type, props) || LayoutFallback
 
-    return (
-      <div>
-        {
-          !page.error && page.loading && PageLoading &&
-          <PageLoading />
-        }
-        {
-          !!page.error && !PageError &&
+    if (page.error) {
+      if (!PageError) {
+        return (
           <div style={ { "text-align": "center" } }>
             <h1>{ page.error }</h1>
             <p>{ page.errorText }</p>
           </div>
-        }
-        {
-          !!page.error && PageError &&
-          <PageError { ...page } />
-        }
-        {
-          !page.error && !page.loading && Layout &&
-          <Layout ref={ this.saveContentRef } { ...page } />
-        }
-      </div>
-    )
+        )
+      }
+      return <PageError { ...page } />
+    }
+    else {
+      if (page.loading && PageLoading) {
+        return <PageLoading />
+      }
+      else if (Layout) {
+        return <Layout ref={ this.saveContentRef } { ...page } />
+      }
+    }
+
+    return null
```

Comment:
Yeah it's totally valid, React render null as... nothing! Handy for component that can skip their render themselves.
But in our case in theory, this should not happen :)